src/index.html

```diff
@@ -7,8 +7,55 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+      href="#"
+      id="hederlink"
+      class="logo">
+        <img
+        src="/images/logo.png"
+        alt="imeg"
+        class="logo_img"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a href="#" class="is-active nav-link link_with_line">apple</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link">samsung</a>
+          </li>
```

Comment:
```suggestion
        <ul class="nav-list">
          <li class="nav-item">
            <a href="#" class="is-active nav-link link_with_line">apple</a>
          </li>

          <li class="nav-item">
            <a href="#" class="nav-link">samsung</a>
          </li>
```
let's add a blank line between sibling elements to increase readability (no need to add between parent and child tags, i.e. after the `ul` tag in this case)